### ISSUE TITLE
file-max-size.js should exclude comments and empty lines

### ISSUE THREAD

**Issue by SemProvoost** [officient/readable#11]
````
$a = '';

// some comment
$b = '';
````
It would be better if this was seen as 2 lines instead of 4

**Comment by denys-potapov** [comment]
Done in master.

**Comment by SemProvoost** [comment]
(not urgent)

if you set include_comments and include-empty lines to true, it still ignores them
e.g. add this as extra test to tests/rules/file-max-size.js:
````
{
      src: src2,
      config: {
        'max-lines': 2,
        'include-comments': true,
        'include-empty-lines': true,
      },
      messageIncludes: 'than 2 lines [6]', // <-- this should be [10], right?
    }
````

this test will pass, no matter how many comment-lines or empty lines I add. -> these added lines shouldn't be ignored, right?
also, src2 has 10 lines. In the test above, I've set it to 6 to make it (wrongly) pass.

**Comment by denys-potapov** [comment]
FIxed in 9b63c02
